pysatSpaceWeather/tests/test_methods_ace.py

```diff
@@ -1,35 +1,105 @@
+#!/usr/bin/env python
+# Full license can be found in License.md
+# Full author list can be found in .zenodo.json file
+# DOI:10.5281/zenodo.3986138
+# ----------------------------------------------------------------------------
+"""Integration and unit test suite for ACE methods."""
+
+from packaging.version import Version
+import pysat
```

Comment:
should pysat be imported lower?

Reply:
It's passing the flake8, so I think it's fine.